packages/node_modules/@webex/plugin-meetings/src/roap/handler.js

```diff
@@ -34,7 +34,7 @@ const checkForAndHandleErrors = (action, meeting, correlationId) => {
   return false;
 };
 
-const compareRemoteRoapOffer = (lastRoapMessage, currentRoapMessage) => currentRoapMessage.messageType === _OFFER_ && lastRoapMessage?.msg?.seq === currentRoapMessage.msg.seq;
+const compareRemoteRoapOffer = (lastRoapMessage, currentRoapMessage) => lastRoapMessage?.msg?.seq === currentRoapMessage.msg.seq && lastRoapMessage?.msg?.messageType === currentRoapMessage.msg.messageType;
```

Comment:
I think the function name should change to compareWithLastRoapMessage() or something like that as it is comparing any message type not just offer